libnopegl/src/glsl/distmap.frag

```diff
@@ -140,14 +140,23 @@ vec2 sum_of_inv(vec2 z0, vec2 z1, vec2 z2)                   { return c_inv(z0 -
 #define XERR 1e-6
 #define SMALL_OFF(off) (dot(off, off) <= XERR * XERR)
 
+float max_3(float a, float b, float c)                   { return max(a, max(b, c)); }
+float max_4(float a, float b, float c, float d)          { return max(a, max_3(b, c, d)); }
+float max_5(float a, float b, float c, float d, float e) { return max(a, max_4(b, c, d, e)); }
```

Comment:
I think yhou can also expose them as helper functions `ngli_max{3,4,5}()`.

Reply:
Removed the commit that drop the helper, and added these functions instead